components/popover/test/index.js

```diff
@@ -227,6 +227,8 @@ describe( 'Popover', () => {
 				style: {},
 			};
 
+			instance.setState = () => {};
```

Comment:
If we assigned this as `jest.fn()` we could add some tests to confirm called with expected behaviors 😄